### ISSUE TITLE
Cookie Banner: Escape `keydown` Event Should Only Set Focus if Cookie Banner is Open

### ISSUE THREAD

**Issue by mpitcel** [app-illinois/Design-Resources#230]
Currently, the Cookie Banner closes if the Escape key is pressed. 
This behavior is expected. 

However, the Escape Key is also triggering this focus change when the Cookie Banner is not open. 
This behavior should be corrected. 

The Escape Key should only change focus if the Cookie Banner is open. 

The Escape `keydown` event is utilized at this point in the JS file: https://github.com/app-illinois/Design-Resources/blob/develop/applications/ila-cookie-banner/ila-cookie-banner.js#L175

**Comment by mpitcel** [comment]
We may be able to rectify this if we update this function (https://github.com/app-illinois/Design-Resources/blob/develop/applications/ila-cookie-banner/ila-cookie-banner.js#L43) to be something like this:

```
function closeCookieB(cookiebId) {
    let cookieb = document.getElementById(cookiebId);

    # Only move forward if the Cookie Banner is Open
    if (cookieb.classList.contains('ila-cookieb--open')) {

        cookieb.classList.remove('ila-cookieb--open');
        cookieb.classList.add('ila-cookieb--closed');

        // Remember that the notice has been dismissed
        setDismissCookieNotice();

        // Used to enable scroll on the page
        document.body.classList.remove('ila-cookieb-noscroll');

        // Used to disable a modal background on the page
        let modalIDvar = document.getElementById('ilaCookieModal');
        modalIDvar.classList.remove('ila-cookieb-modal');

        // Put focus back to the page body on close
        document.body.setAttribute('tabindex', '-1'); // Focusable but outside tab order
        document.body.focus();

    }
}
```
